Bug 1210304 - fixed some David's comments

MariaKolesina opened the pull request:
1) Rather than using the hidden attribute here, it is probably better to set and remove a CSS class that will show the more apps screen on top of the main screen. That way you can specify CSS animations for the transition, which is what our UX designers have in mind, I think.
Done.
2) just use the hidden attribute with no value. No need for the =""
In accordance with previous comment changed to class.
3) If you ever have to use proto your code is almost certainly wrong. Accessing proto is also a big performance hit, since the code can't be optimized by the JS interpreter. You'll have to have a very good reason for proto and a long comment explaining the reason, or I will not land the patch.
Removed proto.
4) To be parallel with the function above, I'd suggest you call this hideMoreApps()
Renamed.

MariaKolesina commented:
@harman-red-square/harman-reviewers

egorlevichev commented:
I would suggest to use 'hidden' for the css class name. Except this, the patch looks good to me. After fixing, please ask AlexeyY to set labels.

MariaKolesina commented:
Thank you, class name has been fixed.

egorlevichev commented:
@AlekoNN please set review labels

AlekoNN commented:
@TatianaMigacheva please merge the changes.

MariaKolesina committed:
Bug 1210304 - fixed some David's comments

1) Rather than using the hidden attribute here, it is probably better to set and remove a CSS class that will show the more apps screen on top of the main screen. That way you can specify CSS animations for the transition, which is what our UX designers have in mind, I think.
Done.
2) just use the hidden attribute with no value. No need for the =""
In accordance with previous comment changed to class.
3) If you ever have to use proto your code is almost certainly wrong. Accessing proto is also a big performance hit, since the code can't be optimized by the JS interpreter. You'll have to have a very good reason for proto and a long comment explaining the reason, or I will not land the patch.
Removed proto.
4) To be parallel with the function above, I'd suggest you call this hideMoreApps()
Renamed.
